fw-child/functions.php

```diff
@@ -100,6 +100,7 @@ function child_theme_enqueue() {
 	) {
 		wp_enqueue_style ( 'wp-block-library' );
 		wp_enqueue_style ( 'gutenberg' );
+		wp_enqueue_style ( 'global-styles' );
```

Comment:
@domcode-ew it seems you can remove `is_singular( 'interactive' )` from the condition, since it is used via resource CPT.

Reply:
No it's not. It's actually used via interactive CPT.
This condition was already existing. I just placed the enqueue in it.

Reply:
@domcode-ew there is no single page for interactive CPT, hence, no need to enqueue anything for that CPT. Removing that part off the condition will improve the old code.